src/nifgen/metadata/nifgen.proto

```diff
@@ -0,0 +1,2150 @@
+
+//---------------------------------------------------------------------
+// This file is generated from NI-FGEN API metadata version 23.0.0
```

Comment:
New exports contain the build number as well, so this is a bit outdated.
Can you check latest?

Reply:
example: https://p4.natinst.com/fsm/perforce/build/exports/ni/nifg/nifgen_grpc_device/official/export/23.0/.latestFinal/metadata/

Reply:
Updated to d41 proto file.